package.json

```diff
@@ -18,6 +18,8 @@
     "@angularclass/conventions-loader": "1.0.13",
     "@angularclass/hmr": "1.2.2",
     "@angularclass/hmr-loader": "3.0.2",
+    "@ngx-translate/core": "^6.0.1",
```

Comment:
Please, use fixed version of the dependencies and install new dependencies through yarn.